docker/Dockerfile.py

```diff
@@ -35,11 +35,14 @@
                          'wget',
                          'curl',
                          'openssh-server',
-                         'mesos=1.0.1-2.0.94.ubuntu1604',
```

Comment:
Don't we still need mesos?

Reply:
We do, but not in ARM, where it's not available. It gets installed on x86 later on.